configs/13.0/packages/gcc/sources

```diff
@@ -22,19 +22,19 @@
 ATSRC_PACKAGE_NAME="GCC (GNU Compiler Collection)"
 ATSRC_PACKAGE_SUBPACKAGE_NAME_0="GNU Standard C++ Library v3 (Libstdc++-v3)"
 ATSRC_PACKAGE_SUBPACKAGE_NAME_1="GNU Libgomp"
-ATSRC_PACKAGE_VER=10.0.0
-ATSRC_PACKAGE_REV=f1f1a94e47cd
+ATSRC_PACKAGE_VER=9.1.1
+ATSRC_PACKAGE_REV=272413
 ATSRC_PACKAGE_LICENSE="GPL 3.0"
-ATSRC_PACKAGE_DOCLINK="https://gcc.gnu.org/onlinedocs/gcc/"
+ATSRC_PACKAGE_DOCLINK="https://gcc.gnu.org/onlinedocs/gcc-${ATSRC_PACKAGE_VER}/gcc/"
 ATSRC_PACKAGE_SUBPACKAGE_DOCLINK_0="https://gcc.gnu.org/libstdc++/"
 ATSRC_PACKAGE_SUBPACKAGE_DOCLINK_1="https://gcc.gnu.org/onlinedocs/libgomp/"
 ATSRC_PACKAGE_RELFIXES=
 ATSRC_PACKAGE_NAMESUFFIX="[C, C++ (g++), fortran, Go]"
 ATSRC_PACKAGE_STR_VER="${ATSRC_PACKAGE_NAME} ${ATSRC_PACKAGE_VER}-ibm-r${ATSRC_PACKAGE_REV}"
 ATSRC_PACKAGE_PRE="test -d gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
-ATSRC_PACKAGE_CO=([0]="git clone https://gcc.gnu.org/git/gcc.git")
-ATSRC_PACKAGE_GIT="git checkout -b gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV} ${ATSRC_PACKAGE_REV}"
-ATSRC_PACKAGE_POST="mv gcc gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
+ATSRC_PACKAGE_CO=([0]="svn co --revision ${ATSRC_PACKAGE_REV} svn://gcc.gnu.org/svn/gcc/branches/ibm/gcc-9-branch" \
+                 [1]="svn co --revision ${ATSRC_PACKAGE_REV} http://gcc.gnu.org/svn/gcc/branches/ibm/gcc-9-branch")
+ATSRC_PACKAGE_POST="mv gcc-9-branch gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
```

Comment:
OK.  We use svn for stable releases.

Also, the changes to configs/next/packages/gcc/sources are OK (but github won't let me comment on it, because it is/was a symlink) and contents are the same (checked with diff between two clones).